tests/cephfs/cephfs_vol_management/cephfs_vol_mgmt_subvolgroup_octal_modes.py

```diff
@@ -0,0 +1,337 @@
+            for subvolumegroup in subvolumegroup_list:
+                if group_name == subvolumegroup["group_name"]:
+                    return subvolumegroup.get("mode")
+
+        log.info("Validate the octal mode set on the subgroup")
+        subgroup_1_mode = get_defined_mode(
+            "subvolgroup_1", subvolumegroup_list=subvolumegroup_list
+        )
+        subgroup_2_mode = get_defined_mode(
+            "subvolgroup_2", subvolumegroup_list=subvolumegroup_list
+        )
+        subgroup_ec_mode = get_defined_mode(
+            "subvolgroup_ec1", subvolumegroup_list=subvolumegroup_list
+        )
+        stat_of_octal_mode_on_kernel_dir1 = fs_util.get_stats(
+            client=clients[0],
+            file_path=kernel_mounting_dir_1.rstrip("/") + subvolgroup1_getpath,
+            format="%a",
+        )
+        stat_of_octal_mode_on_kernel_dir2 = fs_util.get_stats(
+            client=clients[0],
+            file_path=kernel_mounting_dir_2.rstrip("/") + subvolgroup2_getpath,
+            format="%a",
+        )
+        stat_of_octal_mode_on_kernel_dir3 = fs_util.get_stats(
+            client=clients[1],
+            file_path=kernel_mounting_dir_3.rstrip("/") + subvolgroup_ec_getpath,
+            format="%a",
+        )
+        stat_of_octal_mode_on_fuse_dir1 = fs_util.get_stats(
+            client=clients[0],
+            file_path=fuse_mounting_dir_1.rstrip("/") + subvolgroup1_getpath,
+            format="%a",
+        )
+        stat_of_octal_mode_on_fuse_dir2 = fs_util.get_stats(
+            client=clients[0],
+            file_path=fuse_mounting_dir_2.rstrip("/") + subvolgroup2_getpath,
+            format="%a",
+        )
+        stat_of_octal_mode_on_fuse_dir3 = fs_util.get_stats(
+            client=clients[1],
+            file_path=fuse_mounting_dir_3.rstrip("/") + subvolgroup_ec_getpath,
+            format="%a",
+        )
+
+        if int(subgroup_1_mode) != int(stat_of_octal_mode_on_kernel_dir1) and int(
+            subgroup_1_mode
+        ) != int(stat_of_octal_mode_on_fuse_dir1):
+            log.error("Octal values are mismatching on subvolgroup_1")
+            return 1
+        if int(subgroup_2_mode) != int(stat_of_octal_mode_on_kernel_dir2) and int(
+            subgroup_2_mode
+        ) != int(stat_of_octal_mode_on_fuse_dir2):
+            log.error("Octal values are mismatching on subvolgroup_2")
+            return 1
+        if int(subgroup_ec_mode) != int(stat_of_octal_mode_on_kernel_dir3) and int(
+            subgroup_ec_mode
+        ) != int(stat_of_octal_mode_on_fuse_dir3):
+            log.error("Octal values are mismatching on subvolgroup_ec1")
+            return 1
+
+        run_ios(clients[0], kernel_mounting_dir_1)
+        run_ios(clients[0], fuse_mounting_dir_1)
+        run_ios(clients[0], kernel_mounting_dir_2)
+        run_ios(clients[0], fuse_mounting_dir_2)
+        run_ios(clients[1], kernel_mounting_dir_3)
+        run_ios(clients[1], fuse_mounting_dir_3)
```

Comment:
I think it is time to introduce a map to avoid writing the same set of commands with different parameters